docs/src/manual/solutions.md

```diff
@@ -32,6 +32,29 @@ Subject to
  y[b] ≤ 1.0
 ```
 
+## Solutions summary  
+[`show_solution_summary`](@ref) can be used for checking the summary of the optimization solutions.
+
+```jldoctest solutions
+julia> show_solution_summary(model)
+* Solver:GLPK
+
+* Status
+  Termination status : OPTIMAL
+  Primal status      : FEASIBLE_POINT
+  Dual status        : FEASIBLE_POINT
+  Message from the solver:
+  "Solution is optimal"
+
+* Candidate solution
+  Objective value : -205.14285714285714
+  Objective bound : Inf
```

Comment:
We probably want to align these `:` with the line below.

Reply:
I fixed.